### ISSUE TITLE
ValueError: '' does not appear to be an IPv4 or IPv6 address

### ISSUE THREAD

**Issue by slightlyskepticalpotat** [CTFd/CTFd#1686]
**Environment**:

- CTFd Version/Commit: 3.1.1, commit 4c31dc2
- Operating System: Ubuntu 20.04 LTS
- Web Browser and Version: Chromium 85.0.4183.121

**What happened?**

A "500 - Internal Server Error" page showed up after I clicked a user in /admin/users.

**What did you expect to happen?**

I expected to see a user page similar to [this](https://demo.ctfd.io/admin/users/1) (you need to be logged in as admin on the demo ctfd site to see it).

**How to reproduce your issue**

Install CTFd by cloning this, installing dependencies, and using prepare.sh. Then, go to the admin page to manage users and click on one of them. 

**Any associated stack traces or error logs**

Systemctl logs: 
```Oct 14 00:41:45 system-name gunicorn[37980]:     response = IP_ADDR_LOOKUP.get(addr)
Oct 14 00:41:45 system-name gunicorn[37980]:   File "/home/ctfd/.local/share/virtualenvs/CTFd-rOJbThUf/lib/python3.8/site-packages/maxminddb/reader.py", line>
Oct 14 00:41:45 system-name gunicorn[37980]:     (record, _) = self.get_with_prefix_len(ip_address)
Oct 14 00:41:45 system-name gunicorn[37980]:   File "/home/ctfd/.local/share/virtualenvs/CTFd-rOJbThUf/lib/python3.8/site-packages/maxminddb/reader.py", line>
Oct 14 00:41:45 system-name gunicorn[37980]:     address = compat_ip_address(ip_address)
Oct 14 00:41:45 system-name gunicorn[37980]:   File "/home/ctfd/.local/share/virtualenvs/CTFd-rOJbThUf/lib/python3.8/site-packages/maxminddb/compat.py", line>
Oct 14 00:41:45 system-name gunicorn[37980]:     return ipaddress.ip_address(address)
Oct 14 00:41:45 system-name gunicorn[37980]:   File "/usr/lib/python3.8/ipaddress.py", line 53, in ip_address
Oct 14 00:41:45 system-name gunicorn[37980]:     raise ValueError('%r does not appear to be an IPv4 or IPv6 address' %
Oct 14 00:41:45 system-name gunicorn[37980]: ValueError: '' does not appear to be an IPv4 or IPv6 address```

**Comment by ColdHeat** [comment]
You shouldn't really be getting an empty IP. There should generally be an IP in almost every situation. Did you try to manipulate or import data or can you provide an example showing how a user can get an empty IP?

**Comment by ComputerGenius152** [comment]
This happened on a fresh installation of CTFd, and I didn't modify it at all. Could something be wrong with one of the dependencies?

**Comment by TRX-0** [comment]
I am having the same error upon accessing any user on the admin page. Is there any solution to this?

**Comment by TRX-0** [comment]
I figured out what causes the error. If you log into the mysql database and specifically in the `tracking` table there will be user entries without an IP address. Deleting those fixes the issue.

`DELETE FROM tracking WHERE ip='';`

**Comment by slightlyskepticalpotat** [comment]
Closed by #1896
